### DISCUSSION TITLE
How do I get keybinds working for this mpv script?

### DISCUSSION THREAD

**Discussion by etwodev** [iina/iina#3817] [Q&A] [answered]
I have a script called [mpv-webm](https://github.com/ekisu/mpv-webm), and I've been trying to get it working with IINA, I put it inside my scripts folder in config and made sure to enable 'use mpv config directory' at the correct path. It loads the mpv script but the keybind it registers for it doesn't work. Is there any solution to this? The script works fine in mpv.

**Comment by hooke007** [comment, marked as answer]
Try to rebind shortcut via iina instead of script-opts.
```
F1   script-binding mpv_webm/display-webm-encoder
```

**Reply by etwodev** [reply]
Worked perfectly!
Thank you.
